perception_dataset/rosbag2/rosbag2_to_t4_converter.py

```diff
@@ -133,6 +134,8 @@ def convert(self):
         self._annotation_files_generator.save_tables(self._output_anno_dir)
         # Calculate and overwrite num_lidar_prs in annotations
         self._calculate_num_points()
+        if len(self._camera_sensors) > 0:
+            self._create_2d_annotations()
```

Comment:
please add an argument of `--generate-bbox-from-cuboid`, because we don't generate bbox  in some case.